README.md

```diff
@@ -107,7 +107,27 @@ If at some point you see something that could be better, then please open a PR.
 
 `pacman -S --needed base-devel git arch-install-scripts repoctl fuse-overlayfs rsync python-telegram-send`
 
-One must have an active mirror of chaotic-aur running locally and some signing key. Configure them in `/etc/chaotic.conf`, like this:
+One must have an active mirror of chaotic-aur running locally and some signing key. 
```

Comment:
One needs "an active mirror" or a setting (in `/etc/chaotic.conf`) like this:
```
export CAUR_URL='https://builds.garudalinux.org/repos/chaotic-aur/x86_64'
export REPOCTL_CONFIG='/etc/chaotic/repoctl.conf'
export CAUR_REPOCTL_DB_URL="${CAUR_URL}/chaotic-aur.db.tar.zst"
export CAUR_REPOCTL_DB_FILE="/tmp/chaotic/db.tar.zst"
```

And `CAUR_FILL_DEST='https://builds.garudalinux.org/repos/chaotic-aur/pkgs.files.txt'` which is already set by default.